thunder/tests/distributed/test_ddp.py

```diff
@@ -1228,7 +1228,7 @@ def _test_native_ddp_helper(input_data):
     return None
 
 
-def _test_native_fsdp_helper(input_data):
+def _test_native_fsdp_helper(input_data, new_fsdp=True):
```

Comment:
where is paramtrization of `new_fsdp` defined?

Reply:
:grin: I should just use new_fsdp unconditionally. The plan is to switch a first test to fsdp in this PR, and then switch over the other tests and finally remove the old fsdp.